docs/content/sips/index.md

```diff
@@ -4,6 +4,14 @@ date = "2022-03-22T14:53:30Z"
 
 ---
 
-- [Spin Deploy](001-spin-deploy.md)
-- [Application Configuration](002-app-config.md)
-- [Trigger Executors](003-trigger-executors.md)
+A SIP is a Spin Improvement Proposal. It is a design document that describes a
+new feature for Spin. It is intended to be the primary mechanism for proposing
+major work items in the Spin project, and for documenting the design decisions
+that have gone into the project.
+
+Here is a list of accepted SIPs:
+
+- [Spin Deploy](../sips/001-spin-deploy.md)
```

Comment:
Is the `../sips` necessary here? :thinking:

Reply:
It is for how Bartholomew currently does relative paths that are not in the root of the `content/` directory.
(will open an issue to track that in Bartholomew)